packages/pigment-react/src/index.ts

```diff
@@ -4,3 +4,4 @@ export { default as keyframes } from './keyframes';
 export { generateAtomics, atomics } from './generateAtomics';
 export { default as css } from './css';
 export { default as createUseThemeProps } from './createUseThemeProps';
+export { clsx } from 'clsx';
```

Comment:
Is this intentional?

Reply:
It was, but moving it to next PR as that'll be using it.